Storage/Storage/Tools/StorageType+Extensions.swift

```diff
@@ -320,6 +320,33 @@ public extension StorageType {
         return firstObject(ofType: ProductReview.self, matching: predicate)
     }
 
+    /// Retrieves all of the stored ProductVariation's for the provided siteID and productID.
+    /// Sorted by dateCreated, descending
+    ///
+    func loadProductVariations(siteID: Int64, productID: Int64) -> [ProductVariation]? {
+        let predicate = NSPredicate(format: "siteID = %lld AND productID = %lld", siteID, productID)
```

Comment:
What made you decide to use a long long integer?

Reply:
I'm actually going to p2 about the Int64 vs. Int usage soon, was asking Aaron about it yesterday - we store the IDs as Int64, but use it as Int in the codebase. what if the ID (esp. site ID) goes over the max value 2,147,483,647? also since we already store them as Int64, I think it'd be nice to use the original type without manual type casting